cpp/src/community/louvain.cu

```diff
@@ -15,7 +15,17 @@
  */
 
 #include <community/louvain.cuh>
+
+// Disable louvain(experimenta::graph_view_t,...)
+// versions for GPU architectures < 700
+//(this is because cuco/static_map.cuh would not
+// compile on those)
+//
```

Comment:
Suggest adding a "FIXME" here reminding us to delete this after RAPIDS drops support for Pascal.

Reply:
Done.